ansible-tests/validations/no-op-firewall-nova-driver.yaml

```diff
@@ -0,0 +1,15 @@
+---
+- hosts: compute
+  vars:
+    metadata:
+      name: Verify NoOpFirewallDriver is set in Nova
+      description: >
+        When using Neutron, the `firewall_driver` option in Nova must be set to
+        `NoopFirewallDriver`.
+  tasks:
+  - name: Verify the `firewall_driver` value
+    # TODO(shadower): if/when we have more validations that read a value from an
+    # openstack config file, write a fact-gathering module for it.
+    command: 'grep "^\s*firewall_driver\s*=\s*nova\.virt\.firewall\.NoopFirewallDriver\s*$" /etc/nova/nova.conf'
```

Comment:
I suggest you match against word boundaries with "\b" after NoopFirewallDriver, just in case someone puts a comment in its config file.